### PR TITLE
Use juggler@3 and loopback@3 for testing

### PR DESCRIPTION
**Pull request by Amir-61**
Created a `loopback-2.x` branch; however I do not have admin access to make it protected; @superkhau I assume you have the permission; could you please make it protected? Thanks!

/to: @superkhau

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,8 +27,8 @@
     "chai": "^3.2.0",
     "eslint": "^2.7.0",
     "eslint-config-loopback": "^1.0.0",
-    "loopback": "^2.21.0",
-    "loopback-datasource-juggler": "^2.39.0",
+    "loopback": "^3.0.0-alpha.5",
+    "loopback-datasource-juggler": "^3.0.0-alpha.7",
```
Comment (superkhau) [inline package.json]: alpha.8

**Comment by superkhau** [comment]
loopback-2.x branch protected

**Comment by superkhau** [comment]
LGTM, do the fixup and merge. No more review needed.

File: package.json [hunk 2]
```diff
@@ -27,8 +27,8 @@
     "chai": "^3.2.0",
     "eslint": "^2.7.0",
     "eslint-config-loopback": "^1.0.0",
-    "loopback": "^2.21.0",
-    "loopback-datasource-juggler": "^2.39.0",
+    "loopback": "^3.0.0-alpha.5",
+    "loopback-datasource-juggler": "^3.0.0-alpha.8",
```
Comment (bajtos) [inline package.json]: Important: with loopback v3, juggler is a no longer a peer dependency, thus it should not be listed in dependencies of loopback apps/projects.

@Amir-61 could you please fix this in a new PR?

**Pull request merged**